src/data/real/ennreal.lean

```diff
@@ -829,11 +829,19 @@ by simpa only [inv_inv] using @inv_le_inv a⁻¹ b
 @[simp] lemma inv_lt_one : a⁻¹ < 1 ↔ 1 < a :=
 inv_lt_iff_inv_lt.trans $ by rw [inv_one]
 
-lemma top_div : ∞ / a = if a = ∞ then 0 else ∞ :=
-by by_cases a = ∞; simp [div_def, top_mul, *]
-
 @[simp] lemma div_top : a / ∞ = 0 := by simp only [div_def, inv_top, mul_zero]
 
+@[simp] lemma top_div_coe : ∞ / p = ∞ := by simp [div_def, top_mul]
+
+lemma top_div_of_ne_top (h : b ≠ ⊤) : ∞ / b = ∞ :=
```

Comment:
Most of the lemmas around are using `a` as the variable, why are you using `b` here? Same thing for the next lemma. Otherwise, lgtm

Reply:
No reason.
```suggestion
lemma top_div_of_ne_top (h : a ≠ ⊤) : ∞ / a = ∞ :=
```